thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -201,7 +201,6 @@ export default Component.extend({
     'entities',
     'aggregates',
     function () {
-      const { entities, aggregates } = this.getProperties('entities', 'aggregates'); // poll observer
```

Comment:
Is this safe to remove this? I believe @apucher was intentionally triggering a cp recomputation here.